File: app-components/src/dev/java/org/synsystems/onlypass/components/DevApp.java
import org.synsystems.onlypass.components.App;
import org.synsystems.onlypass.components.AppComponent;
import org.synsystems.onlypass.components.DaggerDevAppComponent;

public class DevApp extends App {
  @Override
  protected AppComponent createAppComponent() {
    return DaggerDevAppComponent.create();
  }
}

File: app-components/src/dev/java/org/synsystems/onlypass/components/DevApp.kt
import org.synsystems.onlypass.components.App
import org.synsystems.onlypass.components.AppComponent
import org.synsystems.onlypass.components.DaggerDevAppComponent

class DevApp : App() {

  @Override
  protected fun createAppComponent(): = DaggerDevAppComponent.create()
}